generator/uscore/templates/sequence.rb.erb

```diff
@@ -1,31 +1,99 @@
 # frozen_string_literal: true
 
 require_relative './data_absent_reason_checker'
+require_relative './profile_definitions/<%=name.downcase%>_definitions'
 
 module Inferno
   module Sequence
     class <%=class_name%> < SequenceBase
-      include Inferno::DataAbsentReasonChecker
+      include Inferno::DataAbsentReasonChecker  
+      include Inferno::USCore310ProfileDefinitions
 
       title '<%=title%> Tests'
 
-      description 'Verify that <%=resource%> resources on the FHIR server follow the US Core Implementation Guide'
+      description 'Verify support for the server capabilities required by the <%=profile_name%>.'
+
+      details %(
+        # Background
+
+        The US Core #{title} sequence verifies that the system under test is able to provide correct responses
+        for <%= resource %> queries.  These queries must contain resources conforming to <%= profile_name %> as specified
+        in the US Core v3.1.0 Implementation Guide. <% if resource == 'Device' %>If the system under test contains Device
+        resources that are not implantable, and therefore do not conform to the US Core Implantable Device profile,
+        the tester should provide an Implantable Device Code to the test to ensure that only the appropriate device types
+        are validated against this profile.<% end %>
+
+        # Testing Methodology
+
+        <% if resource == 'Provenance' %>
+        Previously run sequences store references to US Core Provenance resources that are associated with other US Core
+        resources using the appropriate `_revincludes` search.  This set of tests uses these found resources to verify
+        support for the `read` operation.  Each of these resources must conform to the US Core Provenance profile.
+        <% elsif delayed_sequence %>
+        Because <%=resource%> resources are not present o not exist in USCDI, no searches are performed on this test sequence. Instead, references to
```

Comment:
> present o not
typo

Also, do we want these tests to be specific to US Core rather than the program requirements?